csrankings-t.csv

```diff
@@ -357,6 +357,7 @@ Thomas Courtade,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/
 Thomas D. LaToza,George Mason University,https://cs.gmu.edu/~tlatoza,200KNLAAAAAJ
 Thomas D. Nielsen,Aalborg University,http://people.cs.aau.dk/~tdn,6fWF0CgAAAAJ
 Thomas Dreier,Karlsruhe Institute of Technology,http://www.zar.kit.edu/591.php?ID=3,NOSCHOLARPAGE
+Thomas Durieux,TU Delft,https://durieux.me/,CoUKVbUAAAAJ

```

Comment:
No indication of Delft affiliation.

Reply:
Thomas Durieux has joined TU Delft CS (software engineering group recently from KTH)
https://se.ewi.tudelft.nl/people.html